README.md

```diff
@@ -2,7 +2,8 @@
 
 [![IRC Network](https://img.shields.io/badge/irc-freenode-blue.svg "IRC Freenode")](https://webchat.freenode.net/?channels=sddm)
 
-SDDM is a modern display manager for X11 and Wayland aiming to be fast, simple and beautiful.
+SDDM is a modern display manager for X11 (with experimental Wayland support) aiming to
```

Comment:
This sounds like it's also experimental to start Wayland sessions, which is not the case